conf/core/ConfigOptions.asciidoc

```diff
@@ -2105,13 +2220,17 @@ into Hootenanny.
 * Data Type: list
 * Default Value:
 ** `hoot::BuildingMatchCreator`
-** `hoot::ScriptMatchCreator,PoiGeneric.js`
+** `hoot::ScriptMatchCreator,Poi.js`
 ** `hoot::HighwayMatchCreator`
-** `hoot::ScriptMatchCreator,LinearWaterway.js`
+** `hoot::ScriptMatchCreator,River.js`
 ** `hoot::PoiPolygonMatchCreator`
 ** `hoot::ScriptMatchCreator,Area.js`
 ** `hoot::ScriptMatchCreator,Railway.js`
 ** `hoot::ScriptMatchCreator,PowerLine.js`
+** `hoot::ScriptMatchCreator,Point.js`
```

Comment:
Here's the new algs added to the end of the chain. They will only conflate features that aren't identified as match candidates by any other routine.